code/modules/client/preference/loadout/loadout_shoes.dm

```diff
@@ -7,6 +7,14 @@
 	display_name = "sandals, wooden"
 	path = /obj/item/clothing/shoes/sandal
 
+/datum/gear/shoes/winterboots
+	display_name = "winter boots"
+	path = /obj/item/clothing/shoes/winterboots
```

Comment:
Well, we already have the winter coat, which is way more powerful than the boots. Plus, my next PR adds a pulse rifle to the system, so the boots will be irrelevant.

You can merge this no problem.